### ISSUE TITLE
Test Blacklight index_presenter

### ISSUE THREAD

**Issue by awead** [projecthydra/sufia#2345]
### Descriptive summary

We're using Blacklight's `index_presenter` and ought to have a minimal feature test that ensures it's doing what we need it to do. Namely, this would include:

* verifying certain index fields are displayed as facetable links

Other?

**Comment by mjgiarlo** [comment]
See past integration testing here: https://github.com/projecthydra/sufia/blob/821fa7111865272b8130dccd62f9ce05e0d6a975/spec/helpers/blacklight_helper_spec.rb

**Comment by mjgiarlo** [comment]
And discussion here: https://github.com/projecthydra/sufia/issues/2341

**Comment by jcoyne** [comment]
I don't think we want to test the index_presenter. That is already well tested.  If you want to do integration testing, you must call sufia code, not Blacklight code, otherwise there is no integration.

**Comment by atz** [comment]
- That our helper_methods actually receive back from Blacklight arguments that we can handle.  One relies on `arg[:config][:field_name]`, for example.  
- `itemprop`
- `field.label`
- All of our helper_methods: `iconify_auto_link`, `link_to_search`, `index_field_link`, `rights_statement_links`, `human_readable_date`
- That our generated configuration achieves the correct composite behavior.

I agree w/ Justin that an integration test is warranted, but I'll take the old helper test as a provisional measure.

**Comment by atz** [comment]
Also check fields' `if` and `unless` clauses.
